extensions/eclipse-che-theia-plugin-remote/src/node/plugin-remote-init.ts

```diff
@@ -99,6 +105,15 @@ export class PluginRemoteInit {
 
         this.pluginReaderExtension = inversifyContainer.get(PluginReaderExtension);
 
+        // Modify 'configStorage' objects path, to use current user home directory
+        // in remote plugin image '/home/theia' doesn't exist
+        const originalStart = PluginManagerExtImpl.prototype.$start;
+        PluginManagerExtImpl.prototype.$start = async function (params: PluginManagerStartParams): Promise<void> {
```

Comment:
This looks like a hack: if the storage path is not in fact determined from the parameters, we should refactor the plugin API to make the  storage path the responsiblity of the plugin runner. 
I understand that we're trying to get this in urgently, so we do a solution in che-theia, but could we open an issue on the theia repo to do the refactoring?